.cm/summary_table.cm

```diff
@@ -122,12 +124,12 @@ platforms:
       - 'subprojects/internal-integ-testing/'
       - 'subprojects/internal-performance-testing/'
       - 'subprojects/internal-testing/'
-  - cc:
-    name: 'configuration_cache'
+  - configuration:
+    name: 'configuration'
```

Comment:
❓ this seems to be inconsistent with other `core/` labels, like `core_execution` and `core_runtime`.

Reply:
Fixed. See the other comment.